### DISCUSSION TITLE
[Feature Request]: Server name/ip in the results

### DISCUSSION THREAD

**Discussion by balajeek** [alexjustesen/speedtest-tracker#685] [Ideas]
Great program and i love it.  My current setup i have dual internet ISP handled by router in load balanced.  So when the speedtest is done it could be from my 1st ISP Optimum or 2nd from T-mobile Home Internet.  I wish if the one of the columns in the results list are ISP so i can sort and see rather than opening each to see if its Optimun or T-mobile.

Also not sure if its possible if i could have the graph with two lines based on ISP.

**Comment by alexjustesen** [comment]
I think someone requested this before and I like the idea. I run a backup ISP as well so it would be nice to have for sure 👍

**Reply by alexjustesen** [reply]
Found it... #513
